blitzstudios/sleeper-mini
# Adding avatar samples

### Description
Adding avatar samples 

### How To Test

1. View Fetch Sample
2. Open Rosters
3. Open Transactions
4. You should see league avatars, user avatars, player avatars and team avatars. Colors of placeholder avatars should be the same color that the user sees in the rest of the app.
